### PR TITLE
761 unaccepted community (connects #761)

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

File: app/MyApp/app/views/ManageCommunity.js [hunk 1]
```diff
@@ -28,7 +28,9 @@ $(function() {
             var htmlreferance = this.$el
 
             this.$el.append('<div style="padding: 20px 20px 0px 20px; float: left;"> <a id="configuration" href="#configurationsForm"><button class="btn btn-primary" id="configbutton">' + App.languageDict.get('Configurations') + '</button></a> </div>')
-            this.$el.append('<div style="padding: 20px 20px 0px 0px; float: left;"> <button id="sync" class="SyncDbSelect btn btn-primary">' + App.languageDict.get('Sync_With_Nation') + '</button>  </div>')
+            if(App.configuration.get('registrationRequest')=='accepted'){
+                this.$el.append('<div style="padding: 20px 20px 0px 0px; float: left;"> <button id="sync" class="SyncDbSelect btn btn-primary">' + App.languageDict.get('Sync_With_Nation') + '</button>  </div>')
+            } else {}
```
Comment (Pr0chin) [inline app/MyApp/app/views/ManageCommunity.js]: else {} should be removed

**Review by Pr0chin — CHANGES_REQUESTED** [review]

**Review by Pr0chin — APPROVED** [review]

**Review by dogi — COMMENTED** [review]
@razu9861 awesome

File: app/MyApp/app/views/Dashboard.js [hunk 2]
```diff
@@ -1345,13 +1345,12 @@ $(function() {
...
             that.vars.new_survey_count = new_surveys_count;
             this.vars.type = App.configuration.get("type");
-            that.$el.html(_.template(this.template, this.vars));
-            this.checkAvailableUpdates(member.get('roles'), this, nation_version);
-            $('#newPublication').html(App.languageDict.attributes.Publications + '(' + new_publications_count + ')');
-            $('#updateButton').html(App.languageDict.attributes.Update_Available + '(' + nation_version + ')');
-            $('#newSurvey').html(App.languageDict.attributes.Surveys + '(' + new_surveys_count + ')');
-            return new_publications_count;
-
+                that.$el.html(_.template(this.template, this.vars));
```
Comment (lmmrssa) [inline app/MyApp/app/views/Dashboard.js]: fix indenting

File: app/MyApp/app/views/ManageCommunity.js [hunk 3]
```diff
@@ -28,7 +28,9 @@ $(function() {
             var htmlreferance = this.$el
 
             this.$el.append('<div style="padding: 20px 20px 0px 20px; float: left;"> <a id="configuration" href="#configurationsForm"><button class="btn btn-primary" id="configbutton">' + App.languageDict.get('Configurations') + '</button></a> </div>')
-            this.$el.append('<div style="padding: 20px 20px 0px 0px; float: left;"> <button id="sync" class="SyncDbSelect btn btn-primary">' + App.languageDict.get('Sync_With_Nation') + '</button>  </div>')
+            if(App.configuration.get('registrationRequest')=='accepted'){
+                this.$el.append('<div style="padding: 20px 20px 0px 0px; float: left;"> <button id="sync" class="SyncDbSelect btn btn-primary">' + App.languageDict.get('Sync_With_Nation') + '</button>  </div>')
+            }
             this.$el.append('<div style="padding: 20px 20px 0px 0px; float: left;"> <a id="publication" class="btn btn-primary" href="#publications/for-' + App.configuration.get('name') + '">' + App.languageDict.get('Publications') + '</a>  </div>')
```
Comment (lmmrssa) [inline app/MyApp/app/views/ManageCommunity.js]: these also needs to be only when accepted

File: app/MyApp/app/Router.js [hunk 4]
```diff
@@ -924,6 +927,7 @@ $(function() {
         getNationVersion: function (dashboard) {
             var that = this;
             var configuration = App.configuration
+            var jsonModel = getRequestDocFromLocalDB();
```
Comment (lmmrssa) [inline app/MyApp/app/Router.js]: where is this code used?

**Review by lmmrssa — APPROVED** [review]

**Pull request closed without merging**